infra/tf/vars.tf

```diff
@@ -66,7 +70,17 @@ variable "meshdb_networkrange" {
   default      = "24"
 }
 
+variable "meshdb_net_block" {
```

Comment:
TODO: Investigate this further

Reply:
I think this is a variable we use for FRR config.

Reply:
Cleaned up var names